apps/rush-lib/src/logic/base/BaseInstallManager.ts

```diff
@@ -392,7 +393,7 @@ export abstract class BaseInstallManager {
       const committedPnpmFilePath: string = this._rushConfiguration.getPnpmfilePath(this._options.variant);
       const tempPnpmFilePath: string = path.join(
         this._rushConfiguration.commonTempFolder,
-        RushConstants.pnpmfileFilename
+        path.basename(committedPnpmFilePath)
```

Comment:
Would it make more sense to refactor `getPnpmfilePath` in `RushConfiguration` to return the filename instead of the whole path so we don't need to peel the path back apart here?